Rebus.AdoNet/AdoNetSagaPersisterAdvanced.cs

```diff
@@ -0,0 +1,399 @@
+#if true
```

Comment:
We can remove this now that this is completed.

Reply:
ack